src/components/StaticContent/Partners.res

```diff
@@ -38,11 +38,11 @@ let cardStyle = {
 }
 let logoStyle = {
   open Css
-  style(list{marginLeft(#percent(10.)), width(#percent(80.))})
+  style(list{margin(#percent(10.)), width(#percent(80.))})
 }
-let centerText = {
+let corporatePartnerTextStyle = {
   open Css
-  style(list{textAlign(#center)})
+  style(list{textAlign(#center), marginBottom(#percent(10.))})
```

Comment:
I'm slowly moving the code to use 
```rescript
open CssJs
style(.[])
```

instead of:
```rescript
open Css
style(list{})
```

(the api allows us to use arrays instead of lists now)